app/models/user.rb

```diff
@@ -1,2 +1,4 @@
 class User < ApplicationRecord
+  has_many :events
+  has_many :attendees
```

Comment:
The association with events should consider a creator_id as a foreign_key  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from Odin)

A user should have many attended_events, not attendees  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/attendee_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))